fixtures/datasources/database_backup.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: canaries.flanksource.com/v1
+kind: Canary
+metadata:
+  name: database-backup-example
+spec:
+  interval: 60
+  databaseBackup:
+    - maxAge: 3600
```

Comment:
Apologies for noticing it this late in the review... but I think we need to take `string` kind here of type duration and parse it to convert it to seconds by code.... e.g: `5m30s`